pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -229,7 +229,12 @@ func (m *kubeGenericRuntimeManager) GetPods(all bool) ([]*kubecontainer.Pod, err
 	if err != nil {
 		return nil, err
 	}
-	for _, s := range sandboxes {
+	for i := range sandboxes {
```

Comment:
I don't think it matters in this case whether you do or not really in this case. I suspect it might be defensive programming due to the surprising behaviour of https://play.golang.org/p/rRCAgE_Wnz ... but since we don't take the address of 's' anywhere and it's already a pointer, it really doesn't matter either way imo.